common/gen_bash_dns_funcs

```diff
@@ -0,0 +1,887 @@
+#!/bin/bash
+# 2011/10/08 This file may be sourced in order to define basic dns/ip functions like afs_cell, dns_cell, valid_ip, etc.
+
+
+# It is the user program's responsibility to source the following files whose functions are used in this file.
+# gen_bash_print_funcs
+# gen_bash_valid_funcs
+# gen_bash_list_funcs
+
+
+# This creates a variable named <source file base name>_owner and assigns it the value of the owner.
+eval ${BASH_SOURCE##*/}_owner=\"Michael Walsh\"
+# Reference this variable as shown in the following examples.
+# local var_name=${BASH_SOURCE##*/}_owner
+# print_var "${var_name}"
+# echo "${!var_name}"
+
+# 2015/05/13 Mike Walsh (xzy0065) - I moved afs_cell to gen_bash_afs_funcs.  Most of my programs don't automatically source gen_bash_afs_funcs.  Therefore, I'll source it here on their behalf.
+if ! type afs_cell >/dev/null 2>&1 ; then
+  if [ "${HOSTNAME}" = "w3.rchland.ibm.com" -o "${HOSTNAME}" = "w3dev.rchland.ibm.com" ] ; then
```

Comment:
Things like this really don't make much sense in OS env.  Is all this afs stuff in these files really needed?